operator-common/src/main/java/io/strimzi/operator/common/model/PemTrustSet.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.common.model;
+
+import io.fabric8.kubernetes.api.model.Secret;
+import io.strimzi.operator.common.Util;
+
+import java.nio.charset.StandardCharsets;
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+import java.util.Base64;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * Class to represent the set of trusted certificates used by TLS clients
```

Comment:
I don't think there's anything here which is specific to clients

```suggestion
 * Represents the set of certificates to be trusted by a TLS client or server
```